.github/workflows/shield.yml

```
@@ -34,7 +34,6 @@ jobs:
       - uses: actions/checkout@v5
       - uses: actions/setup-go@v5
         with:
-          go-version: "1.24"
+          go-version: "1.25"
       - name: Test (-race)
-        run: go test -race -v ./shield/...
-
+        run: go test -race -v ./shield/...
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)